spec.html

```diff
@@ -31047,6 +31067,93 @@ <h1>AdvanceStringIndex ( _S_, _index_, _unicode_ )</h1>
             1. Return _index_ + _cp_.[[CodeUnitCount]].
           </emu-alg>
         </emu-clause>
+
+        <emu-clause id="sec-getstringindex" aoid="GetStringIndex">
+          <h1>GetStringIndex ( _S_, _Input_, _e_ )</h1>
+          <p>The abstract operation GetStringIndex takes arguments _S_ (a String), _Input_ (a List), and _e_. It performs the following steps when called:</p>
+          <emu-alg>
+            1. Assert: _Input_ contains the code points of _S_ interpreted as a UTF-16 encoded string.
```

Comment:
IIRC, this was pretty much a direct copy of spec text at the time the proposal was drafted, but the underlying spec text has changed since. The ES2016 spec has the following text in [`sec-pattern`](https://262.ecma-international.org/7.0/#sec-pattern):

```
sequence of code points of _str_ interpreted as a UTF-16 encoded (<emu-xref href="#sec-ecmascript-language-types-string-type"></emu-xref>) Unicode string
```

I appear to have dropped the word `Unicode` in my transcription. That said, if the editors believe this is a better interpretation I'm perfectly fine with it.